### DISCUSSION TITLE
Why can't I change the api key value?

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [FlowiseAI/Flowise#156] [General]
I can create an API key but can't decide what value should have! What am I missing?

**Comment by Light1nTheDark** [comment]
same to you ,did you solve this problem？

**Comment by Kvkthecreator** [comment]
![image](https://github.com/FlowiseAI/Flowise/assets/134989386/174a8e51-3eca-414a-8cd4-bfe704ef5b29)

**Reply by Kvkthecreator** [reply]
is there where you're mentioning? FYI, it's not available for embed.
